spark/dl/src/test/scala/com/intel/analytics/bigdl/python/api/PythonSpec.scala

```diff
@@ -127,12 +127,13 @@ class PythonSpec extends FlatSpec with Matchers with BeforeAndAfter {
       valRdd = data.toJavaRDD(),
       vMethods = util.Arrays.asList("Top1Accuracy", "Loss"))
 
-    val trainSummary = TrainSummary(sc.appName, "lenet")
+    val logdir = com.google.common.io.Files.createTempDir()
```

Comment:
Not a warning, but without this it generates its test output in the source tree and git complains it's an untracked file